app/assets/stylesheets/components/_border.scss

```diff
@@ -9,3 +9,7 @@
 .rounded-lg { border-radius: $border-radius-lg; }
 .rounded-xl { border-radius: $border-radius-xl; }
 .rounded-xxl { border-radius: $border-radius-xxl; }
+
+@media #{$breakpoint-sm} {
+  .sm-border-none { border: 0; }
```

Comment:
not sure how this adds more padding...but I guess I'll add that to the list of "Things in CSS that don't make sense to me" :)